File: p11044.java
package UVa;
import java.util.*;
public class p11044 {
    public static void main(String[] args)
    {
        Scanner sc=new Scanner(System.in);
        int n=0;
        n=sc.nextInt();
        while(n>=1)
        {
            n--;
            int row=0,col=0;
            row=sc.nextInt();
            col=sc.nextInt();
            int result=0;
            row=row/3;
            col=col/3;

            result=(row*col);
            System.out.println(+result);
        }

        sc.close();

    }
}

File: p11044.cpp
#include<iostream>
using namespace std;
int main()
{
    int row,col;
    int n;
    cin>>n;
    while(n--)
    {
           cin>>row>>col;
           int result;
           result=((row/3)*(col/3));
           cout<<result<<endl;
    }
 
}